include/rbs/rbs_location.h

```diff
@@ -4,25 +4,16 @@
 #include "lexer.h"
 
 #include "rbs/util/rbs_constant_pool.h"
+#include "rbs/util/rbs_allocator.h"
 #include "rbs/rbs_location_internals.h"
 
 typedef struct rbs_location {
     range rg;
     rbs_loc_children *children;
 } rbs_location_t;
 
-rbs_location_t *rbs_location_new(range rg);
```

Comment:
I'd prefer keeping the `rbs_location_new` naming.

I think you were aiming for consistency with the `alloc_parser`/`alloc_lexer()` functions, but I think we actually want to rename those instead to `rbs_parser_new()`/`rbs_location_new()`